src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -124,8 +124,8 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
                 throw new \RuntimeException(sprintf('Invalid pattern found in "%s" on line "%d"', $ignoreFile, 1 + $line).substr($m, 12));
             });
             try {
-                foreach (file($ignoreFile) as $line => $pattern) {
-                    if ('#' !== trim($line)[0] ?? '#') {
+                foreach (file($ignoreFile, FILE_IGNORE_NEW_LINES) as $pattern) {
```

Comment:
The flag should be removed and $line be kept: it's used by reference in the closure above

Reply:
Thanks, done